### PR TITLE
SRCH-6044: Added sitemap extension check

### PR DESCRIPTION
**Pull request by igoristic**
#### Functionality Checks

- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.

- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.

- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:

#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: search_gov_crawler/search_gov_spiders/sitemaps/sitemap_monitor.py [hunk 1]
```diff
@@ -194,13 +194,21 @@ def _fetch_sitemap(self, url: str, depth: int = 0, max_depth: int = 10) -> Set[s
                     for sitemap in root.findall(f"{ns}sitemap"):
                         loc = sitemap.find(f"{ns}loc")
                         if loc is not None and loc.text:
-                            child_urls = self._fetch_sitemap(loc.text, depth + 1, max_depth)
-                            urls.update(child_urls)
-                else:
+                            loc_text = loc.text.strip()
```
Comment (selfdanielj) [inline search_gov_crawler/search_gov_spiders/sitemaps/sitemap_monitor.py:197]: might be good to lower here since you are testing for a lowercase suffix right?

**Review by selfdanielj — APPROVED** [review]
Much better for the usitc.gov sitemap case!  One suggestion below but definitely an edge case.

My only other comment is I wish that there was more test coverage for this code, I was expecting a test change too.
image

What do you think about making a separate ticket to add some more tests for sitemap?
